tac/project-updates/2025/2025-Q3-Hyperledger-Identus.md

```diff
@@ -0,0 +1,93 @@
+- `Neoprism` PRISM DID indexer (we are planning to deprecate the existing `prism-node` component by the end of 2025)
+- Phase 3 of the VDR driver for Identus on the Cardano blockchain
+- Documentation improvements based on the community feedback and the audit of the Technical Writers
+
+---
+
+## Overall Activity in the Past Quarter
+
+The Identus community remains active across the LFDT Discord, Identus Maintainers' weekly community meetings, and GitHub [discussions](https://github.com/hyperledger-identus/hyperledger-identus/discussions)
+Community members continue to ask questions, report issues, and discuss various use cases, as well as discuss the Identus Roadmap. 
+The core team and community contributors actively support these interactions.
+
+Together with the LFDT, the Identus team conducted the workshop at the RareEvo 2025 event.
+The Socious, Identus community, and the Identu team demonstrated the use case by integrating the platform, employing an offline-first strategy, and integrating the SSI capability into the Lace wallet. 
+The link to the recording is [here](https://www.youtube.com/watch?v=vlBsXzNqvDc).
+The workshop went well and attracted other community members. There were several questions about the use cases and Identus' plans for the future, highlighting the community's interest in the Identus project.
+
+---
+
+## Current Plans
+
+The strategic direction remains consistent with the annual report. The main goals are to:
+
+- Continue reducing the maintenance cost and complexity of the Identus Platform by introducing and adopting alternative ways to resolve the DID Prism ([`Neoprism` project](https://github.com/hyperledger-identus/neoprism) and community initiative to use the file system and GitHub as the [indexer](https://github.com/FabioPinheiro/prism-vdr) for the PRISM DID)
+- Increase adoption among developers and organizations
+- Enhance interoperability with other Self-Sovereign Identity (SSI) platforms
+- Continue the documentation improvement and developer tutorials (documentation audit was completed, and the plan for improvements is defined)
+
+Planned features for the upcoming year include:
+- Complete the Cardano Identus VDR initiative (the current progress can be tracked [here](https://milestones.projectcatalyst.io/projects/1300189/milestones/3))
+- Complete the Issuance capabilities in the SDKs
+- OIDC support: OID4VCI, OID4VP, and SIOP 2.0
+- Authentication via the PRISM DID method
+
+Additional features under consideration:
+- Support of JSON-LD Verifiable Credentials
+- Support of mDoc Verifiable Credentials
+- Support of the Verifiable Credentials Data Model v2
+- Support of OpenBadges 3.0 Verifiable Credentials
+- Integration with WebWallets (the Lace Wallet will use the Identus Platform to enable the SSI capabilities)
+
+To align with LF Decentralized Trust's Project Lifecycle Criteria, the team also plans to:
+- Enable the OpenSSF Best Practices Badge
+- Enable the OpenSSF Scorecard Badge
```

Comment:
This was mentioned as an aspiration in your Q2 report as well, and was also requested by the reviewers of your 2025 annual report. I'm just wondering if you are facing any issues with enabling these badges. It's quite straightforward; please see the instructions in https://github.com/ossf/scorecard?tab=readme-ov-file#scorecard-badges and an equivalent commit for the Cacti project in https://github.com/hyperledger-cacti/cacti/commit/cc2f9c58975f72907576c529c2094b330d8b2b92.